netty/src/test/java/io/grpc/netty/ProtocolNegotiatorsTest.java

```diff
@@ -415,7 +415,10 @@ public void from_tls_clientAuthRequire_noClientCert() throws Exception {
         .trustManager(caCert)
         .build();
     Status status = expectFailedHandshake(channelCreds, serverCreds);
-    assertThat(status.getDescription()).isEqualTo("ssl exception");
+    if (!"ssl exception".equals(status.getDescription())) {
+      assertThat(status.getDescription()).startsWith("io exception");
+      assertThat(status.getDescription()).contains("SslHandler");
+    }
```

Comment:
I think it'll be more expressive to keep the original assertion. Now in of linux where we actually expect "ssl exception" message, it will still report `SslHandler` not found. Which can be misleading.

Maybe combine both of them into a regex?

```suggestion
    assertThat(status.getDescription()).containsMatch("(^ssl exception|SslHandler)");
```

Otherwise, we could catch assertion error for `assertThat(status.getDescription()).isEqualTo("ssl exception");`, and then try asserting contains `SslHandler`? So it shows both exceptions.